### PR TITLE
update chromedriver to 130.0.6723.19

### PR DESCRIPTION
**Pull request by chipitsine**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -11,9 +11,9 @@ RUN echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >
 RUN apt-get update && \
   apt-get install -y google-chrome-beta
 
-RUN pip3 install selenium
+RUN pip3 install selenium --break-system-packages
```
Comment (marten-seemann) [inline Dockerfile:14]: Why is this needed?

File: run.py [hunk 2]
```diff
@@ -37,6 +37,8 @@ def get_args():
 options = webdriver.ChromeOptions()
 options.gpu = False
 options.binary_location = "/usr/bin/google-chrome-beta"
+options.add_argument("--enable-logging")
+options.add_argument("--v=1")
```
Comment (marten-seemann) [inline run.py]: What is this for?

File: Dockerfile (hunk 1 above)
Comment (chipitsine) [reply Dockerfile:14]: image

I didn't find better workaround

Comment (chipitsine) [reply Dockerfile:14]: image

I didn't find better workaround

File: run.py (hunk 2 above)
Comment (chipitsine) [reply run.py]: for debug

https://support.google.com/chrome/a/answer/6271282?hl=en

Comment (marten-seemann) [reply run.py]: Let's split this out into a separate PR.

Comment (marten-seemann) [inline run.py]:
```suggestion
```

Comment (chipitsine) [reply run.py]: no problem
